src/pages/MetricDetail.svelte

```diff
@@ -113,132 +115,41 @@
     in
     <a href={`/apps/${params.app}`}>{params.app}</a>
     that
-    {getExpiryInfo(metric.expires)}
+    {getExpiryInfo(metric.expires)}.
```

Comment:
I think we could move this expiry info to the Metadata table. For the metric type, maybe make it a clickable pill to highlight it?

Reply:
wrt expiry, this makes sense now that we have a warning when something actually has expired. I'll make this change.

I think it's worth calling out the metric type explicitly in the header, as it's rather vitally important -- arguably the most important piece of information about a metric aside from the name.